Strain_Tools/strain/models/strain_visr.py

```diff
@@ -60,7 +60,11 @@ def compute_visr(myVelfield, strain_range, inc, distwgt, spatwgt, smoothincs, ex
     subprocess.call(['mv', strain_data_file, tempdir], shell=False);
     subprocess.call(['mv', strain_output_file, tempdir], shell=False);
     print("Success computing strain via Visr method.\n");
-    return [xdata, ydata, rot, exx, exy, eyy];
+
+    # I know visr allows for velocity calculation, but dropping this in here now as a placeholder
+    Ve, Vn = np.nan*np.empty((4,4)), np.nan*np.empty((4,4))
```

Comment:
why shape = (4,4) ?  This is breaking on my machine for visr and delaunay, which use this line

Reply:
@kmaterna I think that is a typo. I may have fixed it on my other machine and not pushed the update. I'll push an update!